src/pages/Congratulations/index.tsx

```diff
@@ -146,29 +194,59 @@ interface StateProps {
   depositKeys: DepositKeyInterface[];
   workflow: WorkflowStep;
 }
-interface DispatchProps {}
-interface Client {
-  header: string;
-  text: string;
-  imgUrl: any;
-  url: routesEnum;
-  linkText: string;
+interface DispatchProps {
+  dispatchTransactionStatusUpdate: DispatchTransactionStatusUpdateType;
 }
+
 type Props = StateProps & DispatchProps & OwnProps;
 
 const _CongratulationsPage = ({
   depositKeys,
   workflow,
+  dispatchTransactionStatusUpdate,
 }: Props): JSX.Element => {
   const [state, setState] = useState({
     amountEth: 0,
     status: 0,
   });
-  const { amountEth, status } = state;
+  const { status } = state;
   const { formatMessage } = useIntl();
-  const currentAPR = calculateEth2Rewards({ totalAtStake: amountEth });
+  const { account, connector }: web3ReactInterface = useWeb3React<
+    Web3Provider
+  >();
+
+  const totalTxCount = depositKeys.filter(
+    key => key.depositStatus !== DepositStatus.ALREADY_DEPOSITED
+  ).length;
+
+  const remainingTxCount = depositKeys.filter(
+    file =>
+      file.depositStatus !== DepositStatus.ALREADY_DEPOSITED &&
+      file.transactionStatus !== TransactionStatus.SUCCEEDED
+  ).length;
+
+  const allTxConfirmed = _every(
+    depositKeys.map(
+      file => file.transactionStatus === TransactionStatus.SUCCEEDED
+    )
+  );
+
+  const actualTxConfirmed = totalTxCount - remainingTxCount;
+
+  const currentAPR = calculateEth2Rewards({ totalAtStake: actualTxConfirmed });
```

Comment:
@wackerow `totalAtStake` be the total ETH balance across all validators (like on the homepage, which we query from Beaconcha.in)?

Reply:
Yep, good catch 👍... that explains the weird APR we were getting at the end.